docker/build-tools/Dockerfile

```diff
@@ -25,7 +25,7 @@
 # Binary tools
 ################
 
-ARG GOLANG_IMAGE=golang:1.16.6
+ARG GOLANG_IMAGE=golang:1.16.7
```

Comment:
Does it make sense to remove the default value here, and only update it in the single build-and-push.sh? Removing the value here means that it must be supplied as a `build-arg`. It would be as part of the script. Anyone building the image from the Dockerfile themselves would have to add it in their pipeline.

Reply:
A better option would be to update the script to only add the `build-arg` if `GOLANG_IMAGE` is set removing the default value from the script. This removes the second value from the script. 

So, if someone does `GOLANG_IMAGE=<desired value> ./build-and-push.sh` it would pass in a `build-arg` with that value overriding the value in the Dockerfile, while not overriding it if not set.

Reply:
I did submit the later change in https://github.com/istio/tools/pull/1669